include/mesh/checkpoint_io.h

```diff
@@ -35,6 +35,15 @@ namespace libMesh
 {
 // Forward declarations
 class Xdr;
+class CheckpointIO;
+
+/**
+ * splitmesh takes the given initialized/opened mesh and partitions it into nsplits pieces or
```

Comment:
I'd prefer an underscore in the name, i.e. `split_mesh()` but don't feel too strongly about it.

Reply:
Agreed on both counts.